apidef/oas/security.go

```diff
@@ -1196,11 +1196,31 @@ func (s *OAS) GetJWTConfiguration() *JWT {
 		return nil
 	}
 
-	for keyName := range s.getTykSecuritySchemes() {
-		if _, ok := s.Security[0][keyName]; ok {
```

Comment:
Here was the panic. The JWT configuration was actually in s.Security[1] in compliant mode.